CefSharp.BrowserSubprocess.Core/JavascriptObjectWrapper.cpp

```diff
@@ -12,8 +12,19 @@ using namespace CefSharp::Internals;
 
 namespace CefSharp
 {
-	void JavascriptObjectWrapper::Bind()
+    void JavascriptObjectWrapper::Bind()
 	{
+        auto objectName = StringUtils::ToNative(_object->JavascriptName);
+
+        if (_object->IsNull) {
+            auto javascriptObject = V8Value.get() ? V8Value->CreateNull() : CefV8Value::CreateNull();        
```

Comment:
Should probably replace all instance calls with the static method on `CefV8Value` for clarity, there's a few historic cases that should be cleaned up.